flowauth/backend/flowauth/admin.py

```diff
@@ -235,6 +236,18 @@ def add_server():
     json["latest_token_expiry"] = datetime.datetime.strptime(
         json["latest_token_expiry"], "%Y-%m-%dT%H:%M:%S.%fZ"
     )
+    if "name" not in json:
+        raise InvalidUsage("Must provide server name", payload={"bad_field": "name"})
+    if len(json["name"]) == 0:
+        raise InvalidUsage("Must provide server name", payload={"bad_field": "name"})
```

Comment:
This is a duplicate message to above. Do you want to mention "non-empty" here?

Reply:
Sorry, wasn't thinking straight... Of course this message makes sense here and is sufficient. ;)